docs/guides/intro-to-tests.md

```diff
@@ -0,0 +1,154 @@
+# Introduction to tests
+
+## What does this cover
+
+This introduction is designed to give you a high-level overview on the basics of testing, which you'll need when you're tackling the problems in your coursework at Code Your Future.
+In this introduction to testing we'll cover the following things:
+
+- Code specification
+- What is a test ?
+- Anatomy of a test ( using Jest )
+- Understanding test feedback ( using Jest )
+
+## Code specification
+
+`createWordCount` 🧮
```

Comment:
I find the "create" naming here a bit confusing. In my head, this function is performing a calculation, more than "creating" something. The object is just the data format for the results.

I wonder if `countWords` would be more intuitive?

Reply:
:+1: Agreed - I think we should stick to "count"